PWGHF/HFC/TableProducer/correlatorD0Hadrons.cxx

```diff
@@ -101,6 +101,8 @@ struct HfCorrelatorD0HadronsSelection {
   using SelCollisions = soa::Join<aod::Collisions, aod::EvSels>;
 
   Preslice<aod::HfCand2Prong> perCol = aod::hf_cand::collisionId;
+  Preslice<aod::Tracks> perCollisionID = aod::track::collisionId;
+  Preslice<aod::McParticles> perTrueCollision = o2::aod::mcparticle::mcCollisionId;
```

Comment:
They are not used anywhere, why do you slice if then the table slice is not used?